.github/CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 <h3>New features since last release</h3>
 
+* Added the `SingleExcitation` two-qubit operation, which is useful for quantum 
+  chemistry applications. [(#1121)](https://github.com/PennyLaneAI/pennylane/pull/1121)
+  
+  It can be used to perform an SO(2) rotation in the subspace 
+  spanned by the states :math:`|01\rangle` and :math:`|10\rangle`. 
+  For example, the following circuit performs the transformation
+  :math:`|10\rangle \rightarrow \cos(\phi/2)|10\rangle - \sin(\phi/2)|01\rangle`:    
+  
+  ```python
+  dev = qml.device('default.qubit', wires=2)
+
+  @qml.qnode(dev)
+  def circuit(phi):
+      qml.PauliX(wires=0)
+      qml.SingleExcitation(phi, wires=[0, 1])
+  ```
+  
+  The `SingleExcitation` operation supports analytical gradients on hardware
```

Comment:
Not entirely sure myself about the difference, suggestion based on usage in [Evaluating analytic gradients on quantum hardware](https://arxiv.org/pdf/1811.11184.pdf).
```suggestion
  The `SingleExcitation` operation supports analytic gradients on hardware
```